### DISCUSSION TITLE
네이버에서 차단되면 푸는 방법은 없나요?

### DISCUSSION THREAD

**Discussion by d6gkyuj** [kimytsc/covid-rest-vaccine-macro#71] [Q&A] [answered]
카카오톡이 너무 불안해보여서 네이버로 하고 싶은데 차단된거 같아요... 0개의 리스트라고만 뜨네요 혹시 해결방법 없을까요>?

**Comment by kimytsc** [comment, marked as answer]
네이버의 차단은 일정 시간이 지나야 풀리기때문에 기다리는 방법 말고는 없습니다..

핸드폰의 데이터(와이파이 끈 상태)로 네이버에 접근한다면 핸드폰에서는 차단이 풀릴 가능성이 있습니다.
